### ISSUE TITLE
Missing crypto dependency

### ISSUE THREAD

**Issue by jeffgca** [fission-suite/wnfs-migration#5]
It looks like a package is missing or has been re-named:

```
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @noble/secp256k1@1.5.1.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
```

To repro:

1. `git clone git@github.com:fission-suite/wnfs-migration.git`
2. `cd wnfs-migration`
3. `npm i`

Looking on npmjs.com, the 1.5.1 version doesn't seem to exist:

https://www.npmjs.com/package/@noble/secp256k1?activeTab=versions

**Comment by jeffgca** [comment]
Sorry, saw the yarn.lock a little late. Here's the yarn output:

```
➜  wnfs-migration git:(main) yarn
yarn install v1.22.17
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
[3/5] 🚚  Fetching packages...
error An unexpected error occurred: "https://registry.yarnpkg.com/@noble/secp256k1/-/secp256k1-1.5.1.tgz: Request failed \"404 Not Found\"".
info If you think this is a bug, please open a bug report with the information provided in "/Users/jeff/code/fission/clones/wnfs-migration/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

**Comment by icidasset** [comment]
Fix released in version `2.0.2`
